packages/extension/src/tree.view.ts

```diff
@@ -398,14 +398,14 @@ class SnippetItem extends Item implements ContextMenu {
     if (snps.length < 1) {
       snps.push(
         new SnippetItem(
-          'Add New Snippet',
+          'Add New Clipboard',
```

Comment:
to stay consistent with above?
```suggestion
          'Add New Clipboard Item',
```